.circleci/config.yml

```diff
@@ -238,6 +276,32 @@ jobs:
           name: Deploy docs to Netlify
           command: netlify deploy --prod -d dist/docs/helm -s ${NETLIFY_SITE_ID}
 
+  publish-rpm:
+    executor: cloudsmith/default
+    steps:
+      - attach_workspace:
+          at: ~/
+      - cloudsmith/ensure-api-key
+      - cloudsmith/install-cli
+      - cloudsmith/publish:
+          cloudsmith-repository: opennms/common
+          package-format: rpm
+          package-distribution: any-distro/any-version
+          package-path: dist/packages/*.rpm
+
+  publish-deb:
+    executor: cloudsmith/default
+    steps:
+      - attach_workspace:
+          at: ~/
+      - cloudsmith/ensure-api-key
```

Comment:
We could consider using CircleCI contexts to populate the env. variables so we don't need to repeat them in every project.